src/framework/vst/internal/vstpluginprovider.cpp

```diff
@@ -43,6 +43,10 @@ VstPluginProvider::VstPluginProvider(const PluginFactory& factory, const ClassIn
     m_impl = std::make_unique<Impl>(factory, info);
 }
 
+VstPluginProvider::~VstPluginProvider()
+{

```

Comment:
This is necessary because of the destruction of `std::unique_ptr<…> m_impl`, which requires the `Impl` to be a complete type, which is only the case in .cpp and not in .h.